src/components/scenes/GettingStartedScene.tsx

```diff
@@ -88,6 +101,22 @@ export const GettingStartedScene = (props: Props) => {
     navigation.navigate('login', {})
   })
 
+  useAnimatedReaction(
+    () => swipeOffset.value,
+    value => {
+      if (value === 5) {
```

Comment:
Now that I think about it. We could get a nicer animation effect if we hook into the "release" event of the gesture in SwipeOffsetDetector. The event handler would have the new offset as a parameter which could be checked instead of checking the `swipeOffset.value`. This would be better.

Reply:
Forget this one. Scope creep